cmd/openapi2proto/main.go

```diff
@@ -21,10 +21,12 @@ func main() {
 }
 
 func _main() error {
-	specPath := flag.String("spec", "../../spec.yaml", "location of the swagger spec file")
-	annotate := flag.Bool("annotate", false, "include (google.api.http) options for grpc-gateway")
+	specPath := flag.String("spec", "../../spec.yaml", "location of the swagger spec file. Defaults to ../../spec.yaml if not set")
```

Comment:
The stdlib actually tacks on the defaults for us by default, no need to add this here:

```
╰─➤  openapi2proto -h                                                                                                                                                                                                                                                                                                     2 ↵
Usage of openapi2proto:
  -annotate
    	include (google.api.http) options for grpc-gateway
  -indent int
    	number of spaces used for indentation (default 4)
  -out string
    	the file to output the result to. Defaults to stdout if not set
  -spec string
    	location of the swagger spec file (default "../../spec.yaml")
```